go/vt/vtgate/executor.go

```diff
@@ -122,6 +123,11 @@ func NewExecutor(ctx context.Context, serv srvtopo.Server, cell, statsName strin
 
 // Execute executes a non-streaming query.
 func (e *Executor) Execute(ctx context.Context, method string, safeSession *SafeSession, sql string, bindVars map[string]*querypb.BindVariable) (result *sqltypes.Result, err error) {
+	span, ctx := trace.NewClientSpan(ctx, "mysql", "executor.Execute")
```

Comment:
My inclination is that `NewClientSpan` doesn't need to exist at this time and omitting the `peer.service` tag seems like it probably shouldn't cause any harm?

Different tracing services have different ways of describing remote endpoints (for example, zipkin: https://godoc.org/github.com/openzipkin/zipkin-go/model#Endpoint).

Rather than trying to figure out how to accommodate them now, seems like we can leave them untagged. The "mysql" here adds no additional information, since it's implied by the span name.

Reply:
I rather leave it in for now. This is the way OpenTracing annotates peer services [1](https://github.com/opentracing/specification/blob/master/semantic_conventions.md) and not a Jaeger specific thing. In our tracing, marking service such as MySQL and Zookeeper as such is helpful, instead of having to infer it from the method that is running.